src/context/auth-context.js

```diff
@@ -0,0 +1,89 @@
+import { createContext, useContext, useState, useReducer } from "react";
+import axios from "axios";
+import { useEffect} from "react";
```

Comment:
This can also be imported in the first line itself

Reply:
thank you for pointing this out ravi, updated changes